packages/kit/src/views/BulkCopyAddresses/pages/BulkCopyAddresses.tsx

```
@@ -440,12 +565,30 @@ function BulkCopyAddresses({
             rules={{
               required: true,
               min: 1,
-              onChange: (e: { target: { name: string; value: string } }) =>
-                handleFormValueOnChange({
-                  name: e.target.name,
-                  value: e.target.value,
-                  intRequired: true,
-                }),
+              onChange: (e: { target: { name: string; value: string } }) => {
+                const value = (e?.target?.value || '').replace(/\D/g, '');
+                const valueNum = new BigNumber(parseInt(value, 10));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)